Essentials/src/messages_cs.properties

```diff
@@ -602,4 +603,4 @@ worthSet=\u00a76Cena nastavena
 year=rok
 years=roky
 youAreHealed=\u00a76Byl/a jsi uzdraven/a.
-youHaveNewMail=\u00a76Mas \u00a7c{0} \u00a76zprav\! Napis \u00a7c/mail read\u00a76 pro precteni.
+youHaveNewMail=\u00a76Mas \u00a7c{0} \u00a76zprav\! Napis \u00a7c/mail read\u00a76 pro precteni.
```

Comment:
I missed another missing newline here.

Reply:
So, to clarify, you want an empty line at the end of the lang files?